src/styles/blocks/card.css

```diff
@@ -0,0 +1,80 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 16px;
+  border: 1px solid rgb(243, 243, 243);
+  border-radius: 5px;
+  font-family: "Roboto", sans-serif;
```

Comment:
Move the font-family to html tag

Reply:
Move font-family: "Roboto", sans-serif;  to html tag